assembly/AbstractInt.ts

```diff
@@ -4,7 +4,7 @@ import { Bytes, BIT_LENGTH } from "./utils/Bytes";
 /** Representation for a Int value in the system. */
 export abstract class AbstractInt<T extends number> implements Codec {
 
-    readonly bitLength: i32;
+    private bitLength: i32;
```

Comment:
Why do we need this to be private? We won't be able to use it in children implementation

Reply:
Once you create new `Int64` f.e we have the function `int64.encodedLength()` and we were seeing the property `int64.bitLength`. I thought that they are actually the same and we do not need to have them both. I decided to make the bitlength private so that it is not visible publicly by others when they create instance of ints/uints. They have the `encodeLength` function for that.